### PR TITLE
Add a lot of various fuzzed crates

### PR DESCRIPTION
**Pull request by 5225225**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -224,21 +234,26 @@ smoltcp | [index out of bounds](https://github.com/m-labs/smoltcp/commit/3f43be8
...
 syn | [Unrecognized literal](https://github.com/dtolnay/syn/issues/897) | libfuzzer | `logic`
 tar-rs | [#23](https://github.com/alexcrichton/tar-rs/issues/23) | afl | `arith`
 tera | [#396](https://github.com/Keats/tera/issues/396) | libfuzzer | `arith`, `logic`
+tera | [unimplemented panic](https://github.com/Keats/tera/issues/657) | libfuzzer | `panic`
 tiff | [index out of bounds](https://github.com/PistonDevelopers/image-tiff/issues/28) | afl | `oor`
 tiff | [infinite loop on malformed input](https://github.com/PistonDevelopers/image-tiff/issues/31) | afl | `loop`
 tiff | [memory exhaustion on malformed input](https://github.com/PistonDevelopers/image-tiff/issues/29) | afl | `oom`
 tiff | [panic on attempt to divide by zero](https://github.com/PistonDevelopers/image-tiff/issues/33) | afl | `arith`
 time | [issue/309](https://github.com/time-rs/time/issues/309) | afl | `panic`, `arith`
+tinytemplate | [beginning more than end on string slicing](https://github.com/bheisler/TinyTemplate/issues/22) | libfuzzer | `oor`
```
Comment (5225225) [inline README.md:256]: Here, this is an index where begin <= end, do we still classify that as an out of range? Or is this just a unspecified panic?

File: README.md [hunk 2]
```diff
@@ -270,7 +287,9 @@ wayland-rs | [#187](https://github.com/Smithay/wayland-rs/pull/187) | libfuzzer
 ws-rs | [arithmetic overflow](https://github.com/housleyjk/ws-rs/pull/179) | libfuzzer | `arith`
 xi-editor | [issue/1303](https://github.com/xi-editor/xi-editor/issues/1303) | afl | `arith`  
 xml-rs | [#93](https://github.com/netvl/xml-rs/issues/93) | afl | `utf-8`
+xml-rs | [arithmetic overflow](https://github.com/netvl/xml-rs/issues/204) | libfuzzer | `arith`
 zip-rs | [arithmetic overflow](https://github.com/mvdnes/zip-rs/issues/40) | libfuzzer | `arith`
+zip-rs | [arithmetic overflow](https://github.com/zip-rs/zip/issues/234) | libfuzzer | `arith`
```
Comment (5225225) [inline README.md:292]: The repo link here has changed (and the crate is called `zip`, not `zip-rs`), should I change both of these?

**Comment by 5225225** [comment]
Also, I was going through the list of crates, the `prost` stack overflow (which is marked as security sensitive) seems to be just a normal recursive stack overflow. If that's security sensitive, should I then mark `toml` / `toml_edit` / `sqlparser` / `ron` as security sensitive? Those *look* to be basically the same bug, just a recursive call when fed bad input.

Also the link to `prost` is wrong, but I'd rather do fixups to existing lines in a new PR.

File: README.md (hunk 1 above)
Comment (Shnatsel) [reply README.md:264]: `oor` is fine

**Comment by Shnatsel** [comment]
Regular recursive stack overflow is not considered a security issue, unless it's on an embedded platform that lacks guard pages. Fixes to the prost entry are quite welcome.

Also, thanks a lot for all this fuzzing - that is quite an impressive work!

**Pull request merged**